### DISCUSSION TITLE
Stereo Separation distance in HoloLens 2

### DISCUSSION THREAD

**Discussion by fmtibaa** [microsoft/MixedRealityToolkit-Unity#10658] [Q&A] [answered]
Hello,
I want to change the stereo separation distance between eyes on HoloLens 2 but i did not find how to do it, has anyone done it before and could help me ?

Thanks for your responses.

**Comment by Zee2** [comment, marked as answer]
Hi @fmtibaa , the stereo separation/IPD is calculated automatically on-the-fly by the eye tracking system on the device. The IPD will be automatically set correctly for the current user's facial structure as the eye tracker determines the relative locations of the eyes.
